### PR TITLE
[Console] increased test code coverage

### PR DESCRIPTION
**Pull request by FrancescoBorzi**
| Q             | A
| ------------- | ---
| Branch?       |  2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| License       | MIT

### CODE REVIEW

**Comment by wouterj** [comment]
👎 I don't think these test test any significant logic or logic that is prone to errors. Setters/getters aren't tested normally as well.

Full code coverage should never be a goal (and never be a metric imo), covering all behaviour of the public API is important.

**Comment by iltar** [comment]
Not all of them are testing getters though, the other tests are actually testing behavior.

**Comment by fabpot** [comment]
I agree with @wouterj. Having more tests is not always a good idea. Testing real logic of course is a must. But we need to keep in mind that we want the Symfony test suite to be also as fast as possible, so useless tests should not be added.

File: src/Symfony/Component/Console/Tests/Event/ConsoleCommandEventTest.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    /** @var \PHPUnit_Framework_MockObject_MockObject|InputInterface */
+    private $input;
+
+    /** @var \PHPUnit_Framework_MockObject_MockObject|OutputInterface */
+    private $output;
+
+    public function setUp()
+    {
+        $this->command = new Command('my_mock_command');
+        $this->input = $this->getMock('Symfony\Component\Console\Input\InputInterface');
```
Comment (jakzal) [inline src/Symfony/Component/Console/Tests/Event/ConsoleCommandEventTest.php]: `getMock()` [is deprecated](https://github.com/sebastianbergmann/phpunit/wiki/Release-Announcement-for-PHPUnit-5.4.0#changes). Use `createMock()` or `getMockBuilder()` instead.

**Comment by jakzal** [comment]
These tests won't add much overhead, and since it's php they all bring some value (although for future contributions I'd indeed recommend focusing on more important behaviour).

File: src/Symfony/Component/Console/Tests/Event/ConsoleCommandEventTest.php (hunk 1 above)
Comment (ShinDarth) [inline src/Symfony/Component/Console/Tests/Event/ConsoleCommandEventTest.php]: done, thanks

**Comment by ShinDarth** [comment]
@jakzal can you point me on more important tests that need to be done?

**Comment by fabpot** [comment]
Re-reading these tests, I prefer to not add them as they really don't bring much value (and I would rewrite them in a stateless way anyway, so not worth the effort). Thanks for contributing.

**Pull request closed without merging**